python/pyngraph/runtime/backend.cpp

```diff
@@ -17,12 +17,20 @@
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
 
+#include "ngraph/pass/pass_config.hpp"
```

Comment:
We should remove this `include` or add another `compile` wrapper function using the config.